src/components/FreeResponseReview/FreeResponseReview.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { 
+  FreeResponseDefaultContainer, 
+  FreeResponseExpandedContainer, 
+  FreeResponseText, 
+  ReadMoreLink, 
+  COLLAPSED_HEIGHT 
+} from "./FreeResponseReview.styles";
+
+export const FreeResponseReview = ({ free_response, previewMode }: {
+  free_response?: string;
+  previewMode?: boolean;
+}) => {
+
+  const [expanded, setExpanded] = React.useState(false);
+  const [isOverflowing, setIsOverflowing] = React.useState(false);
+  const textRef = React.useRef<HTMLDivElement>(null);
+
+  const FreeResponseContainer = ({ children }: { children: React.ReactNode }) => {
```

Comment:
This component could be a callback, or just move it outside the parent component and pass previewMode in as an argument